defaults/main.yml

```diff
@@ -11,7 +11,7 @@ backups_role_config_paths:
 backups_role_assets_paths: []
 backups_role_cron_job_at: '45 3 * * *'
 
-backups_role_restic_repo_name: "{{ ansible_hostname }}"
+backups_role_restic_repo_name: "{{ inventory_hostname_short }}"
```

Comment:
I prefer to name the repo as the entire `inventory_hostname` and not the short one because this short can generate errors if we reuse the same prefix as in `odoo.coopdevs.org` and `odoo.facto.coop`...